### PR TITLE
Update utils_spice.py to use new location of epel

### PR DESCRIPTION
**Pull request by swapnakrishnan2k**
EPEL rpms have been moved to a new location and 
modifying the utils_spice.py file to use this new location
of the EPEL rpm. 

Reviewed-By: Swapna Krishnan <skrishna@redhat.com>

### CODE REVIEW

**Comment by swapnakrishnan2k** [comment]
@ldoktor Could you please take a look and make sure it looks ok?

File: virttest/utils_spice.py [hunk 1]
```diff
@@ -339,20 +339,20 @@ def deploy_epel_repo(guest_session, params):
         else:
             arch = arch[:-1]
         if "release 5" in guest_session.cmd("cat /etc/redhat-release"):
-            cmd = ("yum -y localinstall http://download.fedoraproject.org/"
-                   "pub/epel/5/%s/epel-release-5-4.noarch.rpm 2>&1" % arch)
+            cmd = ("yum -y localinstall https://dl.fedoraproject.org/"
+                   "pub/epel/epel-release-latest-5.noarch.rpm")
```
Comment (clebergnu) [inline virttest/utils_spice.py]: This one is fine.

File: virttest/utils_spice.py [hunk 2]
```diff
@@ -339,20 +339,20 @@ def deploy_epel_repo(guest_session, params):
...
             logging.info("Installing epel repository to %s",
                          params.get("guest_vm"))
             guest_session.cmd(cmd, print_func=logging.info, timeout=90)
         elif "release 6" in guest_session.cmd("cat /etc/redhat-release"):
-            cmd = ("yum -y localinstall http://download.fedoraproject.org/"
-                   "pub/epel/6/%s/epel-release-6-8.noarch.rpm 2>&1" % arch)
+            cmd = ("yum -y localinstall https://dl.fedoraproject.org/"
+                   "pub/epel/epel-release-latest-6.noarch.rpm")
```
Comment (clebergnu) [inline virttest/utils_spice.py]: This one too.

File: virttest/utils_spice.py [hunk 3]
```diff
@@ -339,20 +339,20 @@ def deploy_epel_repo(guest_session, params):
...
             logging.info("Installing epel repository to %s",
                          params.get("guest_vm"))
             guest_session.cmd(cmd, print_func=logging.info, timeout=90)
         elif "release 7" in guest_session.cmd("cat /etc/redhat-release"):
-            cmd = ("yum -y localinstall http://download.bos.redhat.com/"
-                   "pub/epel/7/%s/e/epel-release-7-5.noarch.rpm 2>&1" % arch)
+            cmd = ("yum -y localinstall https://dl.fedoraproject.org/"
+                   "pub/epel/epel-release-latest-7.noarch.rpm")
```
Comment (clebergnu) [inline virttest/utils_spice.py]: And finally this one is also sound.

**Comment by clebergnu** [comment]
Merged. Thanks!

**Comment by ldoktor** [comment]
Thanks, Swapna, I think we should notify our Beijing team, @ypu I hope you don't depend on the original locations. If so, please make it configurable and send your patches...

**Pull request merged**